examples/image-carousel/reducers/ImageCarouselActions.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  init: () => Object({
+    tiles: [{
+      image: 'https://www.w3schools.com/howto/img_nature_wide.jpg',
+      text: 'Nature and sunrise',
+      alt: 'Nature and sunrise'
+    }, {
```

Comment:
I would move tiles out of the actions, so that way you can just have `selected` be the only thing you are tracking (since you are not altering tiles at all).
You wouldn't have length in `onSelectImage`, `onNextImage`, etc... so you would have to move that logic into the viewing components. @JRJurman your thoughts?

Reply:
I agree, if it's possible, I would definitely recommend only storing logic that you anticipate on changing

Reply:
If you want to, you could store the number of tiles in here, just so that there aren't as many changes required for the actions below.